From e274f7bbcac0ef1ccc6ccd7e98fd645793c5a649 Mon Sep 17 00:00:00 2001
From: Peter Robinson <pbrobinson@gmail.com>
Date: Wed, 23 Jan 2019 03:05:22 +0000
Subject: [PATCH 08/13] Add the latest location for EDK2 OVMF firmware location

The Tianocore EDK2 OVMF firmware can now often be found in the
/usr/share/edk2/ovmf location so add this as an extra option.

Signed-off-by: Peter Robinson <pbrobinson@gmail.com>
---
 oz/ozutil.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/oz/ozutil.py b/oz/ozutil.py
index 273a028..989c9d4 100644
--- a/oz/ozutil.py
+++ b/oz/ozutil.py
@@ -1050,6 +1050,8 @@ def find_uefi_firmware(arch):
     elif arch in ['x86_64']:
         uefi_list = [UEFI('/usr/share/OVMF/OVMF_CODE.fd',
                           '/usr/share/OVMF/OVMF_VARS.fd'),
+                     UEFI('/usr/share/edk2/ovmf/OVMF_CODE.fd',
+                          '/usr/share/edk2/ovmf/OVMF_VARS.fd'),
                      UEFI('/usr/share/edk2.git/ovmf-x64/OVMF_CODE-pure-efi.fd',
                           '/usr/share/edk2.git/ovmf-x64/OVMF_VARS-pure-efi.fd')]
     elif arch in ['aarch64']:
-- 
2.20.1

